content/doc/book/system-administration/admin-password-reset.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+= Reset Jenkins administrator password
+
+As a system administrator, you can reset the Jenkins admin password.
+In case you have forgotten or lost the password, this page provides instructions to reset the admin password.
+
+== Reset the administrator password
+
+. Login to your Jenkins controller and edit the Jenkins configuration file (config.xml) inside your Jenkins folder.
+. Look for the **useSecurity** option and change it from **true** to **false**.
+. Save your file and close it.
+. Restart Jenkins service to apply changes. (This time it will not ask for password)
+. On the dashboard, click on **Manage Jenkins** in the navigation pane on the left side of the page.
+. In Manage Jenkins, in the **Security** section you will get **Configure Global Security**. Click on it.
+. In **Security Realm**, click on the checkbox of **Jenkins' own user database** and save it. You will get redirected to the Manage Jenkins page.
+. In **Security**, click on **Manage Users**. 
```

Comment:
Did you watch the video you added at the bottom of the page? It would be beneficial if the steps written down here align with the steps shown in the video.
Currently, that's not the case.

Reply:
@NotMyFault Actually I have used multiple resources to know how to reset the password. So that's why it is not aligning with the video. I used the simplest way in the documentation. The video link that I have included is just for the sake that the YouTube Channel is frequently sharing information about Jenkins usage and contribution. That's pretty much the case.